File: snippets/proxy/create-service/create-service.js
// If you do not already have it installed, you will need twilio-node 
// installed to run this code: https://www.npmjs.com/package/twilio

const accountSid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX';
const authToken = 'your_auth_token';
const client = require('twilio')(accountSid, authToken);

client.proxy.services.create({uniqueName: 'uniqueName'})
                     .then(service => console.log(service.sid))
                     .done();

File: snippets/proxy/create-service/create-service.cs
using System;
using Twilio;
using Twilio.Rest.Proxy.V1.Service;

const string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
const string authToken = "your_auth_token";

TwilioClient.Init(accountSid, authToken);

var service = ServiceResource.Create(uniqueName: "UniqueName");

Console.WriteLine(service.Sid);
